optuna/multi_objective/visualization/_pareto_front.py

```diff
@@ -16,7 +16,9 @@
 
 @experimental("2.0.0")
 def plot_pareto_front(
-    study: MultiObjectiveStudy, names: Optional[List[str]] = None
+    study: MultiObjectiveStudy,
+    names: Optional[List[str]] = None,
+    plot_all: bool = False,
```

Comment:
This is a not strong opinion, but I prefer `include_all_trials` or `include_dominated_trials` as the option name.
What do you think of it?

Reply:
I agree with you. I was not sure that `plot_all` is a good option name. `include_dominated_trials` is supposed to be nicer.

Reply:
👍